src/standalone/Vegetation/autotrophic_respiration.jl

```diff
@@ -0,0 +1,119 @@
+export AutotrophicRespirationParameters, AutotrophicRespirationModel
+
+abstract type AbstractAutotrophicRespirationModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    AutotrophicRespirationParameters{FT<:AbstractFloat}
+
+The required parameters for the autrophic respiration model.
+$(DocStringExtensions.FIELDS)
+"""
+struct AutotrophicRespirationParameters{FT <: AbstractFloat}
+    "Mean leaf nitrogen concentration (kg N (kg C)-1)"
+    nm::FT
```

Comment:
nm = Vcmax25/ne

ne = 8e-4 mol CO2 m−2 s−1 kg C (kg N)−1 for C3 plants (Clark et al., 2011)

Reply:
Then ne is a parameter, nm is computed. 
How do we call ne?

Reply:
It is just a parameter that converts Vcmax25 to mean nitrogen concentration. We need to define it somewhere.